### PR TITLE
Added DELIMITER support to batch input

### PR DESCRIPTION
**Pull request by Hydrocharged**

### CODE REVIEW

File: integration-tests/bats/sql.bats [hunk 1]
```diff
@@ -792,6 +792,51 @@ SQL
     rm -rf doltsql
 }
 
+@test "sql: batch delimiter" {
```
Comment (zachmu) [inline integration-tests/bats/sql.bats:795]: Make sure this works for delimiters embedded in strings, e.g.

```
mysql> delimiter //
mysql> insert into mytable values (4, "//")
    -> //
Query OK, 1 row affected (0.01 sec)
```

Comment (zachmu) [reply integration-tests/bats/sql.bats:795]: If it doesn't work for this use case (not clear if the quote matching code above is good enough), then fine to add a skipped test instead

**Review by zachmu — APPROVED** [review]
LGTM

File: integration-tests/bats/sql.bats (hunk 1 above)
Comment (Hydrocharged) [reply integration-tests/bats/sql.bats:795]: Ohh that's a good idea!

**Pull request merged**